src/runtime/policy/canonical-policy-annotations.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/** Canonical definitions of annotations used for defining policies. */
+export const canonicalPolicyAnnotations = `
+annotation intendedPurpose(description: Text)
+  targets: [Policy]
+  retention: Source
+  doc: 'Description of a policy.'
+
+annotation egressType(type: Text)
+  targets: [Policy]
+  retention: Source
+  doc: 'Type of egress permitted by a policy. Valid values are: Logging, FederatedAggregation.'
+
+annotation allowedRetention(medium: Text, encryption: Boolean)
+  targets: [PolicyTarget]
+  retention: Source
+  allowMultiple: true
+  doc: 'Indicates the conditions under which the target data can be retained. Valid values for the medium are: Ram, Disk.'
+
+annotation maxAge(age: Text)
+  targets: [PolicyTarget]
+  retention: Source
+  doc: 'Indicates the maximum age the target data can be in order to be used. Age is a string like "2days".'
```

Comment:
could you also document what time units we support? currently TTL works with minutes, hours and days.

It's probably sufficient, as we can't truly support granularity of seconds, and months have variable number of days (and it's easy enough to just write 30 / 60 / 90 days).

Reply:
Done